haas/cli.py

```diff
@@ -27,6 +27,8 @@
 
 command_dict = {}
 usage_dict = {}
+MIN_PORT_NUMBER = 0
+MAX_PORT_NUMBER = 65525
```

Comment:
How about `2**16 - 1`? makes it a bit more clear where the number is coming from.